Library/Formula/git-octopus.rb

```diff
@@ -1,21 +1,16 @@
 class GitOctopus < Formula
   desc "Extends git-merge with branch naming patterns"
   homepage "https://github.com/lesfurets/git-octopus"
-  url "https://github.com/lesfurets/git-octopus/archive/v1.0.tar.gz"
-  sha256 "c83ab4aa3d8ec770f4f05619239a7852c871216a108b4e287c13efb10e3db5db"
-
-  head "https://github.com/lesfurets/git-octopus.git"
-
-  bottle do
```

Comment:
Please leave this block intact. We'll refresh the bottles on pull automatically.